sections/product-recommendations.liquid

```diff
@@ -20,8 +20,22 @@
   }
 {%- endstyle -%}
 
-<div class="color-{{ section.settings.color_scheme }} gradient">
+<div class="color-{{ section.settings.color_scheme }} gradient no-js-hidden">
```

Comment:
This feels like it would be a good fit for a landmark. You could make it an `aside` with an `aria-label`, this way assistive tech users would be able to navigate to it easily.

Reply:
This is for the section that shows up below. But you mean for the complementary block right ?

Reply:
I mean the complementary block, yes 👍

Reply:
Interesting 🤔 changed the wrapper to `aside` and it announces automatically as `complimentary` based on the specs. I think I can keep it as is.

One thing though, not related to this change, when using voice over on Safari for some reasons it traps the focus on this block when it's not set to a `collapsible row`. And I'm not sure why, when I try voice over on chrome it works fine. when I try tabbing only it's not trapped and on TalkBack on Android I also don't get trapped on the block.

Reply:
It announces as complementary because of `aside`, which has that role built in. However, if you add an `aria-label`, then that helps give more context and differentiate from other `complementary` sections (although we don't have more of these at the moment).

Here's what it looks like with an `aria-label` (I added it in as "Product recommendations", but it would have to be translated):

https://user-images.githubusercontent.com/22390758/189345513-de49ffb9-b739-4f3f-ac95-c35b55744b5a.mp4